.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,85 @@
+<!--
+
+---------------------------------------------------
+BUG BOUNTY REPORT INFORMATION
+---------------------------------------------------
+
+WARNING! Don't disclose any information for the Raiblocks bug bounty. First, please read the following paragraphs.
+
+In the interest of further improving the security of the network, we have launched the RaiBlocks Bug Bounty Program.
```

Comment:
Please don't reproduce the entire bug bounty info here.  Instead either refer them to the reddit or medium posts.